io.openems.edge.timeofusetariff.api/src/io/openems/edge/timeofusetariff/test/DummyTimeOfUseTariffProvider.java

```diff
@@ -18,13 +18,31 @@ public class DummyTimeOfUseTariffProvider implements TimeOfUseTariff {
 	 */
 	public static DummyTimeOfUseTariffProvider fromHourlyPrices(ZonedDateTime now, Float... hourlyPrices) {
 
-		var quarterlyPrices = new Float[96];
+		var quarterlyPrices = new Float[24];
 
 		for (var i = 0; i < 24; i++) {
 			quarterlyPrices[i] = hourlyPrices[i];
-			quarterlyPrices[i + 1] = hourlyPrices[i];
-			quarterlyPrices[i + 2] = hourlyPrices[i];
-			quarterlyPrices[i + 3] = hourlyPrices[i];
+		}
+
+		return new DummyTimeOfUseTariffProvider(now, quarterlyPrices);
```

Comment:
why dont you directly return `new DummyTimeOfUseTariffProvider(now, hourlyPrices);`?